dist/search-input/search-input.js

```diff
@@ -0,0 +1,35 @@
+/*(c) Copyright 2015 Pivotal Software, Inc. All Rights Reserved.*/
+"use strict";
+
+var _objectWithoutProperties = function (obj, keys) { var target = {}; for (var i in obj) { if (keys.indexOf(i) >= 0) continue; if (!Object.prototype.hasOwnProperty.call(obj, i)) continue; target[i] = obj[i]; } return target; };
+
+var _extends = Object.assign || function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; };
+
+var React = require("react");
+
+var SearchInput = React.createClass({
+  displayName: "SearchInput",
+
+  propTypes: {
+    placeholder: React.PropTypes.string
+  },
+
+  render: function render() {
+    var _props = this.props;
+    var placeholder = _props.placeholder;
```

Comment:
Is this necessary? I thought react naturally passed down standard html attributes?